api_v3/lib/types/KalturaUserEntry.php

```diff
@@ -159,4 +159,68 @@ protected function doFromObject($srcObj, KalturaDetachedResponseProfile $respons
 		parent::doFromObject($srcObj, $responseProfile);
 	}
 
+	public function validateForInsert($propertiesToSkip = array())
+	{
+		parent::validateForInsert($propertiesToSkip);
+		$this->validatePropertyNotNull("entryId");
+		$this->validateEntryId();
+		$this->validateUserID();
+	}
+	
+	public function validateForUpdate($sourceObject, $propertiesToSkip = array())
+	{
+		if($this->entryId !== null)
+			$this->validateEntryId($sourceObject->getId());
```

Comment:
@yossipapi  I don't see why you need to validate entryId or userId, there is an "@insertOnly" tag in the KalturaUserEntry object on both of these properties which prevents their update.

Reply:
I did not notice the part they are insert only I will remove the update validation in this case.
This comment applies on the other related comments as well.